cmd/operator/service.go

```diff
@@ -0,0 +1,92 @@
+package main
```

Comment:
Should I keep this file or just move the content to the `main.go` file?

Reply:
We can move. Doesn't make a lot of difference for now.